### PR TITLE
Fixed Indie88 connector

### PR DESCRIPTION
**Pull request by jbwharris**
**Describe the changes you made**
I discovered the Indie88 connector was broken for both player version I built the connector for. Looks like they just moved around some UI elements on their pages. 

**Additional context**
image

### CODE REVIEW

File: src/connectors/thecurrent.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const filter = new MetadataFilter({
+const filter = MetadataFilter.createFilter({
```
Comment (inverse) [inline src/connectors/thecurrent.js:3]: Can you undo the changes to this file 👍🏼

**Review by inverse — CHANGES_REQUESTED** [review]
Looks good but we should undo changes to `thecurrent.js` 👍🏼

**Comment by jbwharris** [comment]
@inverse I will see if I can sort that later today.

**Comment by jbwharris** [comment]
@inverse Just pushed that change

**Comment by inverse** [comment]
@jbwharris I'm still seeing the changes in the branch. See the number of files changed at the top is 2

image

Looks like your creating the PR from your `master` branch. Perhaps try 

- making your master match this repo master
- branch from that
- apply only the required changes to that branch
- create a PR from that

**Comment by jbwharris** [comment]
@inverse I will get to sorting this out, may take until the weekend though as it's been a busy one at work

**Review by jbwharris — COMMENTED** [review]
@inverse I'm not quite sure what the the change to thecurrent.js is, since I did replace the file with the latest published version from GitHub.

**Comment by inverse** [comment]
The problem I believe is because you're creating the pull requests from the `master` branch of your fork which normally would be fine for a single change.

However a better practice is to create a branch from that and create changes relevant for that change.

Depending on how you set this up you might be able to get this working with:


```bash
git checkout -B fix-indie88
git checkout master -- src/connectors/thecurrent.js
```

and then commit the undo to `src/connectors/thecurrent.js`

**Comment by jbwharris** [comment]
I'm going to close this pull request and see if I can figure it out, as I can't seem to get that second file to pull back

**Comment by inverse** [comment]
No worries - take your time and let me know if you need any further help.

**Pull request closed without merging**